examples/seismic/acoustic/acoustic_example.py

```diff
@@ -66,8 +66,8 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 if __name__ == "__main__":
     description = ("Example script for a set of acoustic operators.")
     parser = ArgumentParser(description=description)
-    parser.add_argument('-nd', dest='ndim', default=3, type=int,
-                        help="Preset to determine the number of dimensions")
+    parser.add_argument("-d", "--shape", default=(51, 51, 51), type=int, nargs="+",
+                        help="Determine the grid size")
```

Comment:
"Determine the grid size" -> "The grid shape"

Reply:
This should be an extra argument with nd overwritten if shape is provided not replacing it.
`-d` is not a very good name IMO, would enforce `--shape`

Reply:
I think the `-d` is for legacy reasons -- `benchmark.py` takes `-d` for the shape, AFAIR. I don't mind keeping it or removing it. Your call guys

Reply:
I did inherit from benchmark. @mloubout I will add it as extra , OK.

Reply:
Added as extra argument.